src/vcpkg/commands.build.cpp

```diff
@@ -774,6 +776,16 @@ namespace vcpkg
             all_features.append(feature->name + ";");
         }
 
+        auto& post_portfile_includes = action.pre_build_info(VCPKG_LINE_INFO).post_portfile_includes;
+        std::string all_post_portfile_includes;
+        if (!post_portfile_includes.empty())
+            all_post_portfile_includes = std::accumulate(
+                std::next(post_portfile_includes.begin()),
+                post_portfile_includes.end(),
+                post_portfile_includes[0].generic_u8string(),
+                [](const Path& a, const Path& b) { return a.generic_u8string() + ";" + b.generic_u8string(); });
```

Comment:
there is probably a besser way to do this with the helpers vcpkg has but I am mostly unfamiliar with them.